Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1354,15 +1354,20 @@ private string GetCommandLineArguments(string outputPath, EncodingOptions encodi
                 segmentFormat = "mpegts";
             }
 
+            var maxMuxingQueueSize = encodingOptions.MaxMuxingQueueSize >= 128
```

Comment:
```suggestion
            var maxMuxingQueueSize = encodingOptions.MaxMuxingQueueSize > 128
```
Minor nitpick, but this way we don't need to convert an int to a string when the value is 128